src/SFA.DAS.AANHub.Application.UnitTests/Employers/Commands/CreateEmployersCommandHandlerTests.cs

```diff
@@ -0,0 +1,32 @@
+﻿using AutoFixture.NUnit3;
+using FluentAssertions;
+using Moq;
+using NUnit.Framework;
+using SFA.DAS.AANHub.Application.Employers.Commands;
+using SFA.DAS.AANHub.Domain.Entities;
+using SFA.DAS.AANHub.Domain.Enums;
+using SFA.DAS.AANHub.Domain.Interfaces;
+using SFA.DAS.AANHub.Domain.Interfaces.Repositories;
+
+namespace SFA.DAS.AANHub.Application.UnitTests.Employers.Commands
+{
+    public class CreateEmployersCommandHandlerTests
+    {
+        [Test, AutoMoqData]
+        public async Task Handle_AddsNewEmployer(
+            [Frozen] Mock<IMembersWriteRepository> membersWriteRepository,
+            [Frozen] Mock<IAuditWriteRepository> auditWriteRepository,
+            [Frozen] Mock<IAanDataContext> aanContext,
+            CreateEmployerCommandHandler sut,
+            CreateEmployersCommand command)
+        {
+            membersWriteRepository.Setup(p => p.Create(It.IsAny<Member>()));
+            auditWriteRepository.Setup(p => p.Create(It.IsAny<Audit>()));
+            aanContext.Setup(p => p.SaveChangesAsync(It.IsAny<CancellationToken>()));
```

Comment:
I think these should be verifies instead of setup

Reply:
I think setup is correct as we're mocking the functions for the test, rather than checking the result

Reply:
Default setup is not required, the test would work without it. I think it is important to verify the steps that handler is going to perform.

Reply:
Set up removed and verify added